proto/store/activity.proto

```diff
@@ -48,6 +48,7 @@ message ActivityIssueCommentCreatePayload {
       STATUS_UNSPECIFIED = 0;
       PENDING = 1;
       APPROVED = 2;
+      SENT_BACK = 3;
```

Comment:
The potential approval status in enums can vary depending on the specific domain or industry you are referring to. However, here are some common examples of approval status enums that you may encounter in various contexts:

Pending: Indicates that something is awaiting approval or is currently under review.
Approved: Denotes that something has been successfully authorized or granted approval.
Rejected: Indicates that something has been declined or denied approval.
In Progress: Suggests that the approval process is ongoing and not yet completed.
Withdrawn: Denotes that a request for approval has been voluntarily withdrawn before a decision was made.
Expired: Indicates that the approval or authorization has lapsed or is no longer valid.
Conditional: Denotes a status where approval has been granted but with certain conditions or stipulations.
Deferred: Indicates that the approval decision has been postponed or delayed for further consideration.
Under Review: Suggests that the request or application is currently being evaluated or assessed.
These are just some examples, and the specific approval statuses can vary depending on the context and the system or organization implementing the enum.


Pending: The approval is still pending and has not yet been reviewed.
Approved: The approval has been reviewed and approved.
Rejected: The approval has been reviewed and rejected.
Reassigned: The approval has been reviewed and rejected, but the request has been reassigned to another approver.
Cancelled: The approval process has been cancelled.
Skipped: The approval process has been skipped, typically because the approval is not required.

Reply:
upvote for `Rejected` since it is simple and a single word is always better than two or more. 
"Send back"/"Sent back" is more like what we name it in the real word.